src/ioc/init.ts

```diff
@@ -0,0 +1,95 @@
+// To organize imports in a better way
+
+import { SERVICES, CONTROLLERS, IntegrationProvider, INTEGRATION_PROVIDER, IS_INITIALIZED } from './types';
+import { iocContainer } from './container';
+
+// Services
+import ConfigService from '#src/services/config/config.service';
+import EpgService from '#src/services/epg/epg.service';
+import AccountCleengService from '#src/services/account/cleeng.account.service';
+import AccountJWService from '#src/services/account/jw.account.service';
+import CheckoutCleengService from '#src/services/checkout/cleeng.checkout.service';
+import CheckoutJWService from '#src/services/checkout/jw.checkout.service';
+import SubscriptionJWService from '#src/services/subscription/jw.subscription.service';
+import SubscriptionCleengService from '#src/services/subscription/cleeng.subscription.service';
+import CleengService from '#src/services/cleeng/cleeng.service';
+import ApiService from '#src/services/api/api.service';
+import EntitlementService from '#src/services/entitlement/entitlement.service';
+import WatchHistoryService from '#src/services/watchHistory/watchhistory.service';
+import FavoritesService from '#src/services/favorites/favorites.service';
+
+// Controllers
+import WatchHistoryController from '#src/controllers/WatchHistoryController';
+import AccountController from '#src/controllers/AccountController';
+import ConfigController from '#src/controllers/ConfigController';
+import FavoritesController from '#src/controllers/FavoritesController';
+import SettingsController from '#src/controllers/SettingsController';
+import EpgController from '#src/controllers/EpgController';
+import ApiController from '#src/controllers/ApiController';
+import EntitlementController from '#src/controllers/EntitlementController';
+import CheckoutController from '#src/controllers/CheckoutController';
+
+const CLEENG_SERVICES = {
+  [SERVICES.Cleeng]: CleengService,
+  [SERVICES.Account]: AccountCleengService,
+  [SERVICES.Checkout]: CheckoutCleengService,
+  [SERVICES.Subscription]: SubscriptionCleengService,
+} as const;
+
+const JW_SERVICES = {
+  [SERVICES.Account]: AccountJWService,
+  [SERVICES.Checkout]: CheckoutJWService,
+  [SERVICES.Subscription]: SubscriptionJWService,
+} as const;
+
+const COMMON_CONTROLLERS = {
+  [CONTROLLERS.WatchHistory]: WatchHistoryController,
+  [CONTROLLERS.Favorites]: FavoritesController,
+  [CONTROLLERS.Epg]: EpgController,
+  [CONTROLLERS.Entitlement]: EntitlementController,
+} as const;
+
+const bindServices = (servicesMap: { [x: symbol]: new (...args: any[]) => object }) => {
+  Object.getOwnPropertySymbols(servicesMap).forEach((key) => iocContainer.bind(key).to(servicesMap[key]));
+};
+
+export const initIOCData = (integration: IntegrationProvider | null) => {
+  if (!iocContainer.isBound(IS_INITIALIZED)) {
+    iocContainer.bind(IS_INITIALIZED).toConstantValue(true);
+  }
+
+  if (integration === INTEGRATION_PROVIDER.CLEENG) {
+    bindServices(CLEENG_SERVICES);
```

Comment:
I would suggest to explicitly call `iocContainer.bind` for each class inline so it's less magic and more readable.  The `bindServices` helper would be nice if we needed to dynamically bind services, but I think it's fine for us to 'hard code' our container compositions.
```suggestion
    iocContainer.bind<CleengService>(SERVICES.Cleeng).to(CleengService);
    iocContainer.bind<AccountCleengService>(SERVICES.Account).to(AccountCleengService);
    iocContainer.bind<CheckoutCleengService>(SERVICES.Checkout).to(CheckoutCleengService);
    iocContainer.bind<SubscriptionCleengService>(SERVICES.Subscription).to(SubscriptionCleengService);
```